### PR TITLE
Add favicons for all devices and browsers

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/laminas/laminas.github.io/issues/175

### CODE REVIEW

**Comment by froschdesign** [comment]
@Arhell 
Thank you very much for your efforts, but unfortunately we have the [same problems again as in the past](https://github.com/laminas/laminas.github.io/pull/105):

- the favicons should be added to the [website repository](https://github.com/laminas/getlaminas.org)
- [each sub-project has its own logo](https://github.com/laminas/getlaminas.org/tree/master/public/images/logo) and should also have its own favicon
- the current favicon [does not work on dark backgrounds](https://github.com/laminas/laminas.github.io/pull/105#pullrequestreview-481174975)

**Comment by Arhell** [comment]
@froschdesign  You're right, I already managed to forget that I also wanted to do this

Updated the icons, added a logo with a white background for the dark theme, but in general the site does not support the dark theme

Thanks

**Comment by froschdesign** [comment]
> Updated the icons, added a logo with a white background for the dark theme, but in general the site does not support the dark theme

The dark theme of the website is not meant here, but the dark theme of a browser.

Please also see my other notes.
Thanks in advance! 👍🏻

**Comment by Arhell** [comment]
Of course, I will do it in the next PRs
If these icons are suitable, is it possible to merge?

**Comment by froschdesign** [comment]
> If these icons are suitable to this repository, is it possible to merge?

Unfortunately no, because no logo or image is to be added to this repository. The central place is: https://github.com/laminas/getlaminas.org/tree/master/public/images/favicon / https://getlaminas.org/images/favicon/…

**Comment by Arhell** [comment]
but as I looked at this PR https://github.com/laminas/laminas.github.io/pull/159 refer specifically to this repository and not to the project as a whole, if I close this PR and add icons here https:// github.com/laminas/getlaminas.org/tree/master/public/images/favicon then they are not added


Here is an example
https://github.com/laminas/laminas.github.io/blob/master/index.html#L9
Screenshot_1

**Comment by Arhell** [comment]
or do you mean add the logos to the central repository and add links from there here?

**Comment by Arhell** [comment]
> > …or do you mean add the logos to the central repository and add links from there here?
> 
> Correct!

Yeah, thanks, then I won't close for now, update the icons there and update the links here.
But there is a question, site.webmanifest may not work in a cross repository, may leave icons for chrome here?

**Comment by froschdesign** [comment]
> But there is a question, site.webmanifest may not work in a cross repository, may leave icons for chrome here?

No icons here and I think the manifest file can be omitted because we are not producing a "web application".
The icons for the sub-projects alone are good and a necessary improvement.

File: components/index.html [hunk 1]
```diff
@@ -6,7 +6,16 @@
...
+
+    <link rel="apple-touch-icon" sizes="180x180" href="https://getlaminas.org/images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://getlaminas.org/images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://getlaminas.org/images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="https://getlaminas.org/images/favicon/site.webmanifest">
+    <link rel="mask-icon" href="https://getlaminas.org/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="https://getlaminas.org/images/favicon/favicon.ico">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="msapplication-config" content="https://getlaminas.org/images/favicon/browserconfig.xml">
+    <meta name="theme-color" content="#ffffff">
```
Comment (froschdesign) [inline components/index.html:18]: The sub-project "Components" has a different logo, so the current favicon is wrong here. See also:

- https://github.com/laminas/laminas.github.io/pull/176#issuecomment-1478986847
- https://github.com/laminas/getlaminas.org/pull/136#issuecomment-1488540683

File: mvc/index.html [hunk 2]
```diff
@@ -5,7 +5,17 @@
...
+
+    <link rel="apple-touch-icon" sizes="180x180" href="https://getlaminas.org/images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://getlaminas.org/images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://getlaminas.org/images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="https://getlaminas.org/images/favicon/site.webmanifest">
+    <link rel="mask-icon" href="https://getlaminas.org/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="https://getlaminas.org/images/favicon/favicon.ico">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="msapplication-config" content="https://getlaminas.org/images/favicon/browserconfig.xml">
+    <meta name="theme-color" content="#ffffff">
```
Comment (froschdesign) [inline mvc/index.html:17]: The sub-project "MVC" has a different logo, so the current favicon is wrong here. See also:

- https://github.com/laminas/laminas.github.io/pull/176#issuecomment-1478986847
- https://github.com/laminas/getlaminas.org/pull/136#issuecomment-1488540683

**Review by froschdesign — CHANGES_REQUESTED** [review]
Some more favicons are needed, as each sub-project has its own logo and should also have its own favicon.

File: components/index.html (hunk 1 above)
Comment (Arhell) [reply components/index.html]: Done

File: mvc/index.html (hunk 2 above)
Comment (Arhell) [reply mvc/index.html]: Done

**Comment by weierophinney** [comment]
@froschdesign Is this ready to merge at this point? I feel I should defer to you, as I'm not entirely clear on what's going on with the favicon set...

**Comment by froschdesign** [comment]
The XML cannot be accessed, but this is a website problem: https://github.com/laminas/getlaminas.org/blob/35903a8465c826cc814e5773d0fb942a386d0fe7/.platform.app.yaml#L82-L85

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
@Arhell 
Thank you for your time, your patience and this contribution! 👍

**Pull request merged**